### PR TITLE
Feature/more tests

### PR DESCRIPTION
**Pull request by hayssams**
## Summary
More tests to increase coverage

### CODE REVIEW

File: src/main/scala/com/ebiznext/comet/utils/SparkJob.scala [hunk 1]
```diff
@@ -26,6 +26,28 @@ trait SparkJob extends StrictLogging {
     */
   def run(): Try[SparkSession]
 
+  // TODO Should we issue a warning if used with Overwrite mode ????
```
Comment (hayssams) [inline src/main/scala/com/ebiznext/comet/utils/SparkJob.scala:29]: We extract the duplicated code into a private method.
This increase considerably the test coverage for this file and also improve code quality

File: src/test/scala/com/ebiznext/comet/job/atlas/AtlasConfigSpec.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.ebiznext.comet.job.atlas
```
Comment (hayssams) [inline src/test/scala/com/ebiznext/comet/job/atlas/AtlasConfigSpec.scala:1]: Let's start make sure that in the tests we are aware of all cli config options

File: src/test/scala/com/ebiznext/comet/job/convert/Parquet2CSVSpec.scala [hunk 3]
```diff
@@ -85,5 +85,25 @@ class Parquet2CSVSpec extends TestHelper {
       csvFile.delete()
       rootDir.delete()
     }
+
```
Comment (hayssams) [inline src/test/scala/com/ebiznext/comet/job/convert/Parquet2CSVSpec.scala:88]: Let's start make sure that in the tests we are aware of all cli config options

File: src/test/scala/com/ebiznext/comet/job/index/bqload/BiqQueryLoadJobSpec.scala [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package com.ebiznext.comet.job.index.bqload
+
+import com.ebiznext.comet.TestHelper
+
```
Comment (hayssams) [inline src/test/scala/com/ebiznext/comet/job/index/bqload/BiqQueryLoadJobSpec.scala:4]: Let's start make sure that in the tests we are aware of all cli config options

File: src/test/scala/com/ebiznext/comet/job/index/jdbcload/JdbcLoadJobSpec.scala [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+          |  --source_file <value>    Full Path to source file
+          |  --output_table <value>   JDBC Output Table
+          |  --driver <value>         JDBC Driver to use
+          |  --partitions <value>     Number of Spark Partitions
+          |  --batch_size <value>     JDBC Batch Size
+          |  --user <value>           JDBC user
+          |  --password <value>       JDBC password
+          |  --url <value>            Database JDBC URL
+          |  --create_disposition <value>
+          |                           Big Query Create disposition https://cloud.google.com/bigquery/docs/reference/auditlogs/rest/Shared.Types/CreateDisposition
```
Comment (mhdkassir) [inline src/test/scala/com/ebiznext/comet/job/index/jdbcload/JdbcLoadJobSpec.scala:22]: How come we used Big Query create and write dispositions in a JDBCLoadConfig

Comment (hayssams) [reply src/test/scala/com/ebiznext/comet/job/index/jdbcload/JdbcLoadJobSpec.scala]: removed

Comment (hayssams) [reply src/test/scala/com/ebiznext/comet/job/index/jdbcload/JdbcLoadJobSpec.scala:22]: Actually, it is used to determine wether the JDBC table should be created or appended act ...

Comment (mhdkassir) [reply src/test/scala/com/ebiznext/comet/job/index/jdbcload/JdbcLoadJobSpec.scala:22]: Yes, perhaps we should consider using something more generic

**Pull request merged**